api/views/tags.py

```diff
@@ -14,5 +17,53 @@ def followed_tags(
     min_id: str | None = None,
     limit: int = 100,
 ) -> list[schemas.Tag]:
-    # We don't implement this yet
-    return []
+    queryset = Hashtag.objects.followed_by(request.identity)
+    paginator = MastodonPaginator()
+    # TODO: this fails due to the Hashtag model not having an `id` field
```

Comment:
Maybe try paginating over the HashtagFollow objects instead?

Reply:
Yeah, that's a good idea. I added a new `FollowedTag` schema which adds the ID from the `HashtagFollow` model, which works well enough 👍